app/src/main/java/net/gsantner/markor/ui/DraggableScrollbarScrollView.java

```diff
@@ -0,0 +1,94 @@
+package net.gsantner.markor.ui;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.MotionEvent;
+import android.view.View;
+import android.widget.ScrollView;
+
+public class DraggableScrollbarScrollView extends ScrollView {
+
+    private boolean _isFastScrolling = false;
+    private boolean _fastScrollEnabled = true;
+    private boolean _ltr = true;
+    private int _thumbHeight;
+
+    public DraggableScrollbarScrollView(Context context) {
+        super(context);
+        setSmoothScrollingEnabled(true);
+        setLtr();
+    }
+
+    public DraggableScrollbarScrollView(Context context, AttributeSet attrs) {
+        super(context, attrs);
+        setSmoothScrollingEnabled(true);
+        setLtr();
+    }
+
+    public DraggableScrollbarScrollView(Context context, AttributeSet attrs, int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+        setSmoothScrollingEnabled(true);
+        setLtr();
+    }
+
```

Comment:
Combine the 2/3 constructors above to just use the `this()` constructor for the remaining one

Reply:
That resulted in odd behavior, and after noticing that inside the constructors, getLayoutDirection() always returns LTR, I moved that call to onAttachedToWindow().